config/sys.config

```diff
@@ -131,6 +131,15 @@
                   {size, 4194303}, {date, "$D0"}, {count, 3}
                ]}
              ]}
+           ]},
+          {aestratum_lager_event, [
+             {handlers, [
+               {lager_file_backend, [
+                  {file, "log/aestratum.log"},
```

Comment:
I understand stratum not meant for deployment in known env so no [integration in datadog](https://github.com/aeternity/infrastructure/blob/cd97ba9c76c0979f763da7bd4111ea3ceb575f79/ansible/monitoring.yml#L216-L235) needed cc @dincho . Ok.